spark-bigquery-tests/src/main/java/com/google/cloud/spark/bigquery/integration/QueryPushdownIntegrationTestBase.java

```diff
@@ -201,4 +201,41 @@ public void testMathematicalFunctionExpressions() {
     assertThat(r1.get(21)).isEqualTo(false); // ISNAN(word_count)
     assertThat(r1.get(22)).isEqualTo(1.0); // SIGNUM(word_count)
   }
+
+  @Test
+  public void testMiscellaneousExpressions() {
+    Dataset<Row> df =
+        spark
+            .read()
+            .format("bigquery")
+            .option("materializationDataset", testDataset.toString())
+            .load(TestConstants.SHAKESPEARE_TABLE);
+    df.createOrReplaceTempView("shakespeare");
+    df =
+        df.selectExpr(
+                "word",
+                "word_count AS WordCount",
```

Comment:
Could you add `SortOrder` as well for completeness if it is trivial enough?

Reply:
Added orderBy in the end

Reply:
Right. But I don't see an assert for it.

Actually, never mind. Till we fix the order by issue, we won't get the right order anyway.